msbuild/Xamarin.iOS.Tasks.Windows/Xamarin.iOS.Common.After.targets

```diff
@@ -28,6 +28,16 @@ Copyright (C) 2011-2013 Xamarin. All rights reserved.
 		DependsOnTargets="_SayHello" 
 		BeforeTargets="_CleanDebugSymbols;_CleanAppBundle;_DetectAppManifest;_DetectSdkLocations;_GenerateBundleName" />
 	
+	<!-- Default context path values for remote builds -->
+	<!-- When building from VS, these values will be passed directly from the IDE -->
+	<Target Name="EnsureMessagingContext" BeforeTargets="_SayHello">
+		<PropertyGroup>
+			<BasePath Condition="'$(BasePath)' == ''">~/Library/Caches/maui/PairToMac</BasePath>
+			<LogsPath Condition="'$(LogsPath)' == ''">~/Library/Logs</LogsPath>
+			<LocalBasePath Condition="'$(LocalBasePath)' == ''">%LOCALAPPDATA%\maui\iOS\PairToMac</LocalBasePath>
```

Comment:
Where are these values read/used?

Reply:
In the Messaging libraries, but we don't want to hardcode values in Messaging, so we set them from either those targets or from VS.
They are mostly used in the SayHello task and SSH connection related logic

Reply:
Does that code handle expanding `~` to $HOME?